# Thruster
int64 throttle
int64 FR
int64 FL
int64 CR
int64 CL
int64 BR
int64 BL
int64 CB

# Gyro
int64 Yaw
int64 Pitch
int64 Roll
int64 Depth

# Gripper
int64 stepper
int64 pneumatic

# PID Mode
int64 PID_DepthMode
int64 PID_StabilizeMode

# PID Monitor
int64 monitor_stp_roll
int64 monitor_stp_pitch
int64 monitor_stp_yaw
int64 monitor_stp_depth

# Reserved Mission Button
int64 Mission1_Mode
int64 Mission2_Mode
int64 Mission3_Status
int64 Mission4_Status 


